build/assets/master/0200_clusterrole.yaml

```diff
@@ -11,4 +11,5 @@ rules:
   verbs:
   - get
   - patch
-  - update
+  - update 
+
```

Comment:
nit: is there a reason for adding these empty lines in the asset files? I understand a newline at the end of the last line, but not necessarily an empty line